package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "axios-service-base-ts",
   "private": false,
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
We will need to bump to `0.3.0` as this breaks the API as you've seen with https://github.com/Xkonti/quasar-clean-typescript/issues/11

Reply:
Update to `0.3.0`